cv-data-controller/src/main/java/com/trihydro/cvdatacontroller/controller/RsuController.java

```diff
@@ -118,7 +118,7 @@ public ResponseEntity<List<WydotRsu>> SelectActiveRsus() {
 		return ResponseEntity.ok(rsus);
 	}
 
-	@RequestMapping(method = RequestMethod.GET, value = "/rsus-for-tim/{timId}")
+	@RequestMapping(method = RequestMethod.GET, produces = "application/json", value = "/rsus-for-tim/{timId}")

```

Comment:
Note: Opening a work item to update all endpoints in the project to explicitly return JSON may be beneficial.